### ISSUE TITLE
AttributeError: 'UndefinedType' object has no attribute 'localizations'

### ISSUE THREAD

**Issue by Marat-BY** [dr-leo/pandaSDMX#204]
Trying to import the pandaSDMX at the project. 
Returns the error message:
```
AttributeError: 'UndefinedType' object has no attribute 'localizations'
```

List of requirements:
----------------------------- ---------
alabaster                     0.7.12
argon2-cffi                   20.1.0
async-generator               1.10
atomicwrites                  1.4.0
attrs                         20.3.0
Babel                         2.9.0
backcall                      0.2.0
bleach                        3.3.0
certifi                       2020.12.5
cffi                          1.14.5
chardet                       4.0.0
codecov                       2.1.11
colorama                      0.4.4
coverage                      5.5
decorator                     4.4.2
defusedxml                    0.6.0
docutils                      0.16
entrypoints                   0.3
idna                          2.10
imagesize                     1.2.0
iniconfig                     1.1.1
ipykernel                     5.5.0
ipython                       7.21.0
ipython-genutils              0.2.0
ipywidgets                    7.6.3
jedi                          0.18.0
Jinja2                        2.11.3
jsonschema                    3.2.0
jupyter                       1.0.0
jupyter-client                6.1.11
jupyter-console               6.2.0
jupyter-core                  4.7.1
jupyterlab-pygments           0.1.2
jupyterlab-widgets            1.0.0
lxml                          4.6.2
MarkupSafe                    1.1.1
mistune                       0.8.4
nbclient                      0.5.3
nbconvert                     6.0.7
nbformat                      5.1.2
nest-asyncio                  1.5.1
notebook                      6.2.0
numpy                         1.20.1
packaging                     20.9
pandas                        1.2.2
pandaSDMX                     1.4.1
pandocfilters                 1.4.3
parso                         0.8.1
pickleshare                   0.7.5
pip                           21.0.1
pluggy                        0.13.1
prometheus-client             0.9.0
prompt-toolkit                3.0.16
py                            1.10.0
pycparser                     2.20
pydantic                      1.8
Pygments                      2.8.0
pyparsing                     2.4.7
pyrsistent                    0.17.3
pytest                        6.2.2
pytest-cov                    2.11.1
python-dateutil               2.8.1
pytz                          2021.1
pywin32                       300
pywinpty                      0.5.7
pyzmq                         22.0.3
qtconsole                     5.0.2
QtPy                          1.9.0
requests                      2.25.1
requests-cache                0.5.2
requests-mock                 1.8.0
Send2Trash                    1.5.0
setuptools                    54.0.0
six                           1.15.0
snowballstemmer               2.1.0
Sphinx                        3.5.1
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        1.0.3
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.4
terminado                     0.9.2
testpath                      0.4.4
toml                          0.10.2
tornado                       6.1
traitlets                     5.0.5
typing-extensions             3.7.4.3
urllib3                       1.26.3
wcwidth                       0.2.5
webencodings                  0.5.1
widgetsnbextension            3.5.1

**Comment by dr-leo** [comment]
Please provide the trace back indicating the source of the error. Otherwise this exception is not reproducible

**Comment by Marat-BY** [comment]
![image](https://user-images.githubusercontent.com/43601384/109781004-9709e700-7c18-11eb-90f0-92d3d8a2b31a.png)

**Comment by ki-terhoeven** [comment]
Seems to be an issue with a pydantic update.
```pip install pydantic==1.7```
did the trick for me.

**Comment by dr-leo** [comment]
Thanks. This problem might be related to difficulties in declaring minimum versions of dependencies which depend on the python version in use. When using python 3.9, the minimum pydantic version is 1.7.

**Comment by paologentleman** [comment]
I am still getting the same exception with both versions of pydantic (1.7 and 1.8.1). Currently using python 3.7.3.

**Comment by mephinet** [comment]
I can confirm that adding `pydantic<1.8` to my requirements.txt solves the issue.

**Comment by dr-leo** [comment]
Thanks for spotting and suggesting a solution. It is fixed in v1.4.2 which is available on pypi and conda-forge.
